.github/workflows/backport.yml

```diff
@@ -210,7 +210,7 @@ jobs:
         if: ${{ steps.create_backport_pull_request.outputs.has-conflicts == 'false' }}
         uses: juliangruber/approve-pull-request-action@v1
         with:
-          github-token: ${{ secrets.METABASE_AUTOMATION_USER_TOKEN }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
While this fixes backports created via comment, I think this will fail backports created via the tag, right?

Reply:
I was under the (possibly mistaken) impression that this workflow was only used for the comment-based backports and that the label-based backports were handled by auto-backport.yml, which [also uses GITHUB_TOKEN](https://github.com/metabase/metabase/blob/master/.github/workflows/auto-backport.yml#L88) for the approve PR step.

Is this workflow triggered from contexts other than the comment-based creation?

Reply:
you're right! 🫠